### PR TITLE
FEAT: Create the communication parsing class

### PR DESCRIPTION
**Pull request by Peralban**

### CODE REVIEW

File: communication.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            return 1
+
+    def turn_cmd(self, args: list[str]) -> int:
+        if len(args) != 2:
+            return 1
+        try:
+            x : int = int(args[0])
+            y : int = int(args[1])
+            if 0 <= x < self.map_size and 0 <= y < self.map_size:
+                self.queue.put(("enemi", x, y))
```
Comment (Thomaltarix) [inline communication.py:31]:
```suggestion
                self.queue.put(("enemy", x, y))
```

File: communication.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                print("us 0 0")
```
Comment (Thomaltarix) [inline communication.py:32]: What is that for ?

File: communication.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    def error_cmd(self, args: list[str]) -> int:
+        if args:
+            return 1
+        print("ERROR")
+        return -1
+
+    def about_cmd(self, args: list[str]) -> int:
+        if args:
+            return 1
+        print('name="brain", version="0.0", author="oui-oui"')
```
Comment (Thomaltarix) [inline communication.py:59]: Author "oui-oui" ?

**Review by Thomaltarix — CHANGES_REQUESTED** [review]
Two little requests about little changes to do, and a question 👍

File: communication.py (hunk 1 above)
Comment (Peralban) [reply communication.py:31]: We need to discuss what I add in the queue

File: communication.py (hunk 2 above)
Comment (Peralban) [reply communication.py:32]: Temporary here there will are a print for our move.

**Review by Thomaltarix — APPROVED** [review]
Put a description to your pull-request please, but great

**Review by AubaneNourry — APPROVED** [review]

**Pull request merged**